### DISCUSSION TITLE
How to export a json of the character with Animation Debug?

### DISCUSSION THREAD

**Discussion by SerapioDeAlicante** [BoloVEVO/Kade-Engine#37] [Q&A]
I noticed this version of kade uses json data for characters but the character editor can't export a json of the character

lol

**Comment by BoloVEVO** [comment]
For now the character editor doesn't count with a functional exporting character function, instead you will have to write the entire json file for the character.  You can use as a template from the vanilla character jsons the game already has in data/characters.

I know, doing that is very struggling to do but I'll be working on it to make it easier soon.
